modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/ValueProvidersParameterDeclarationEnricher.java

```diff
@@ -255,6 +312,21 @@ private void enrichParameter(OfValueInformation ofValueInformation,
     return optionResolverEnabledParameters;
   }
 
+  private Map<ParameterDeclaration, List<FieldValues>> introspectParameterFields(List<ParameterDeclaration> parameters) {
+    Map<ParameterDeclaration, List<FieldValues>> optionResolverEnabledParameters = new HashMap<>();
+
+    parameters.forEach(param -> {
+      List<FieldValues> fieldValuesList = new ArrayList<>();
+      getAnnotation(param, FieldsValues.class)
```

Comment:
Isn't checking for FieldsValues enough ? aren't you putting one annotation twice in the list?

Reply:
if there is only one annotation getting  FieldsValues.class won't work: https://docs.oracle.com/javase/tutorial/java/annotations/repeating.html